app/views/layouts/application.html.erb

```diff
@@ -50,6 +50,12 @@
               <button id="butInstall" class="material-icons mdc-top-app-bar__action-item mdc-icon-button">cloud_download</button>
             </span>
             <%= yield :credits %>
+            <div class="mdc-menu-surface--anchor" data-controller='user-menu'>
+              <button class="material-icons mdc-top-app-bar__action-item mdc-icon-button" aria-label="Open user menu" data-user-menu-target='trigger' data-action='click->user-menu#open'>
```

Comment:
```suggestion
              <button class="material-icons mdc-top-app-bar__action-item mdc-icon-button" aria-label="Open user menu" data-user-menu-target='trigger' data-action='user-menu#open'>
```

i think `click` is the default action for buttons

Reply:
Updated! [aa44c4f](https://github.com/cedarcode/mi_carrera/pull/300/commits/aa44c4fb7aecd1cc6d4bdd24b9f9ed0de2527aa5)